presto-docs/src/main/sphinx/connector/hive-caching.rst

```diff
@@ -137,7 +143,8 @@ In all cases, you should avoid using the root partition and disk of the node and
 instead attach at multiple dedicated storage devices for the cache on each node.
 The cache uses the drive up to a configurable percentage. Storage should be
 local on each coordinator and worker node. The directory needs to exist before
-Presto starts.
+Presto starts. We recommend using two directories to improve performance of the
```

Comment:
Why two, specifically? Is this two directories per disk, or is it trying to say that multiple disks can improve performance? What if you only have one cache disk, should you still create two directories on it?

Reply:
Add clarification